### PR TITLE
Create help command

### PR DESCRIPTION
**Pull request by tiewweijian**
Things that have been added in this commit

* New content in the Help Window
* New hyperlink to User Guide in Help Window

The text in the Help Window are currently placeholders and will be updated accordingly when the functions are created.

### CODE REVIEW

**Comment by tiewweijian** [comment]
Sorry, my bad I forgot to create test cases on this commit. Will create a new commit !

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,8 +15,13 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY2122S2-CS2103-F11-2/tp/blob/master/docs/UserGuide.md";
+    public static final String HELP_MESSAGE = "Available functions on this version: \n\n"
+            + "-add n/NAME id/STUDENT_ID c/COURSE \n-delete STUDENT_ID\n"
```
Comment (LeeEnEn) [inline src/main/java/seedu/address/ui/HelpWindow.java:21]: Perhaps the `add` command should include all the parameters.

**Review by LeeEnEn — APPROVED** [review]
Minor changes needed to the `add` command, otherwise LGTM!

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (domlimm) [reply src/main/java/seedu/address/ui/HelpWindow.java:21]: WJ, could also add a spacing between the `-` and `TEXT` in your next code PR

**Pull request merged**